src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/ChangePasswordSteps.java

```diff
@@ -0,0 +1,74 @@
+package org.openmrs.contrib.isanteplus.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.isanteplus.qaframework.RunTest;
+import org.openmrs.contrib.isanteplus.qaframework.automation.test.TestBase;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.AccountsPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.HomePage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.LoginPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.ManageAccountsPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.SystemAdministrationPage;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+
+public class ChangePasswordSteps extends TestBase {
+
+	private LoginPage loginPage;
+
+	private SystemAdministrationPage systemPage;
```

Comment:
i would make systemAdministrationPage as a variable

Reply:
@sherrif10 ,What do you exactly mean ?

Reply:
```suggestion
	private SystemAdministrationPage systemAdministrationPage;;
```
Something like